### PR TITLE
Add ChowDSP software

### PR DESCRIPTION
**Pull request by jatinchowdhury18**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +80,7 @@ Open Source Audio Apps
 | FlexASIO | [dechamps/FlexASIO](https://github.com/dechamps/FlexASIO) | Universal ASIO driver for Windows |
 | [Guitarix](https://guitarix.org/) | [SourceForge → guitarix](https://sourceforge.net/projects/guitarix/) | GNU/Linux Virtual Amplifier |
 | [Helio Workstation](https://helio.fm/) | [helio-fm/helio-workstation](https://github.com/helio-fm/helio-workstation) | Sequencer |
+| [NewMixer](https://github.com/jatinchowdhury18/NewMixer) | [jatinchowdhury18/NewMixer](https://github.com/jatinchowdhury18/NewMixer) | Mixing Software |
```
Comment (webprofusion-chrisc) [inline README.md]: Can you move this to the apps section, the plugins are supposed to be items that embed in a host app. Thanks!

Comment (jatinchowdhury18) [reply README.md]: Hi Chris,

I believe this is in the apps section, no? (picture from my fork attached)
OpenAudio

Comment (webprofusion-chrisc) [reply README.md]: My mistake, was reading it on my phone!

**Comment by webprofusion-chrisc** [comment]
Thanks!

**Comment by Marat-Tanalin** [comment]
Is CHOW an abbreviation (where each letter expands to a word)?

**Comment by jatinchowdhury18** [comment]
@Marat-Tanalin nope, just the name of the plugin.

**Comment by Marat-Tanalin** [comment]
So could it probably be written as Chow? Just like Sony is SONY on its logo while actually being Sony.

Unreasonable uppercase is typically considered undesirable practice.

**Comment by webprofusion-chrisc** [comment]
@marat-tanalin for branding it's up to the creator to decide uppercase/lowercase/titlecase. I thought it was CHOW make it sound a bit like a comic book POW :)

**Comment by Marat-Tanalin** [comment]
Yeah, that’s why I’m asking.

**Comment by jatinchowdhury18** [comment]
Yeah, as long as I'm not breaking any rules, I'd prefer to leave it as CHOW

**Pull request merged**